README.md

```diff
@@ -53,7 +59,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+The likely memory requirement for a parser that can validate a particular path configuration is O(1), as the only memory needed to iterate over the tiles is one or two moving pointers, with no extra allocated memory.
```

Comment:
# -3

It has to push half of the input onto the stack in order to match the other half
in reverse order, so it is O(n)